### ISSUE TITLE
unintuitive user-end splitIds requirement is a hindrance

### ISSUE THREAD

**Issue by cerpintext** [slaurent22/hk-split-maker#44]
One quick example given here with "Lemm2" (named like that in the autosplitter to not break compatibility with older split files) which is displayed as "Lemm Shop (NPC)" in the actual user-facing ui in livesplit. I wonder if this could possibly be amended so the front-end ui works with those names, then changed in the back-end to the splitId values?

![image](https://user-images.githubusercontent.com/27957371/135315943-c8f32337-05b0-4b5d-a45a-4ec4024e2109.png)

**Comment by slaurent22** [comment]
agreed. when we end up overhauling the ui, I'm going to want to add a search/autocomplete feature into the tool

**Comment by cerpintext** [comment]
overhauling ax2uhappy i'll keep the issue open so it doesnt slip out of mind when design and work on that actually starts 👍
